app-admin/syslog-summary/syslog-summary-1.14-r5.ebuild

```diff
@@ -1,9 +1,9 @@
-# Copyright 1999-2023 Gentoo Authors
+# Copyright 1999-2024 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=7
+EAPI=8
```

Comment:
Going through the ebuild and double checking with diffing the enviroment. I can say this is a safe change to do without a revbump.